lib/scripts/controllers/options.js

```diff
@@ -97,6 +99,7 @@ export class OptionCtrl {
       this.blacklistedWords = data.userBlacklistedWords.slice(1, -1).split('|');
       this.blacklistedWebsites = data.blacklist.slice(1, -1).split('|');
       this.userDefinedTranslations = JSON.parse(data.userDefinedTranslations);
+      this.userDefinedTranslationList = this.toList(this.userDefinedTranslations);
```

Comment:
@Ceilican the List is for the view. Pagination requires things to be stored as a list and not as an object. Hence I created a duplicate. This does make it prone to bugs, but the other option was to change the entire storage structure of user defined translations.